docs/users-guide/13-sql-parameters.md

```diff
@@ -161,8 +138,165 @@ WHERE True
+
+Field filter variables should be used inside of a `WHERE` clause in SQL, or a `$match` clause in MongoDB.
+
+### Field filter compatible types
+
+Field filters ONLY work with the following field types:
+
+- Category
+- City
+- Entity Key
+- Entity Name
+- Foreign Key
+- State
+- UNIX Timestamp (Seconds)
+- UNIX Timestamp (Milliseconds)
+- ZIP or Postal Code
+
+The field can also be a datetime, which can be left as "No semantic type" in the data model.
+
+When you set the __Variable type__ to "Field Filter", Metabase will present an option to set the __Field to map to__, as well as the __Filter widget type__. The options available for the Filter widget type depend on the field's type. For example, if you map to a field of type Category, you'll see options for either "Category" or None. If you map to a Date Field, you'll see options for None, Month and year, Quarter and year, Single date, Date range, or Date filter.
+
+If you don't want a widget on the question at all, which you might do, for example, if you just want to allow this question to be mapped to a dashboard filter (see more on that below), you can set the __Filter widget type__ to "None".
+
+If you're not seeing the option to display a filter widget, make sure the mapped field is set to one of the above types, and then try manually syncing your database from the "Databases" section of the Admin Panel to force Metabase to scan and cache the field's values.
+
+If you want to map a Field Filter to a field that isn't one of the compatible types listed above, you'll need an Admin to change the [field type for that column][column-types]. 
+
+## Field filter syntax
+
+The syntax for Field Filters differs from a Text, Number, or Date SQL variable.
+
+```
+SELECT count(*)
+FROM products
+WHERE {% raw %}{{date_var}}{% endraw %}
+```
+
+Note the lack of an `=`. The reason you need to structure field filters this way is to handle cases where Metabase generates the code for you. For example, for handling situations where someone selects multiple values in the filter widget, or a range of dates.
+
+
+A MongoDB native query example might look like this:
+
+```
+{% raw %}[ {$match: {{date_var}} } ]{% endraw %}
+```
+
+## Creating SQL question filters using Field Filter variables
+
+Let's say you want to create a field filter that filters the `People` table by state, and you want people to be able to select multiple states at a time. Here's the query:
+
+```
+SELECT *
+FROM PEOPLE
+WHERE {%raw%}{{state}}{%endraw%}
+```
+
+Then, in the side panel, select the "Field Filter" variable type, and choose which field to map your variable to (in this case, `State`).
+
+For a more in-depth guide, check out [Field Filters: create smart filter widgets for SQL questions][field-filter].
+
+## Field types that don't work with Field Filter widgets
```

Comment:
Could this section just be a comma-separated listed and added to the bottom of "Field filter compatible types" section?